src/ipc/ipc4/helper.c

```diff
@@ -283,11 +283,6 @@ int ipc_comp_connect(struct ipc *ipc, ipc_pipe_comp_connect *_connect)
 	return IPC4_INVALID_RESOURCE_STATE;
 }
 
-/* when both module instances are parts of the same pipeline Unbind IPC would
```

Comment:
@tmleman can I ask you a question regarding the unbinding of modules, lets you have a pipeline like this: 
![tplg](https://user-images.githubusercontent.com/7766921/153498980-93b2febf-69a8-46ba-b9d2-a687367e969b.png)


And pipeline 12 is stopped while pipeline 1 is still active. Should we unbind mixin12.1 and mixout.2.1 before pipeline 12 is deleted? Would it cause any issues because mixout.2.1 is in active state still?

Reply:
I made some research and its was my mistake to assume that unbind of modules under the same pipeline should be normally processed.
Documentation informs that this operation should be ignored, so error reporting via IPC is also incorrect.
@RanderWang will returning 0 and adding a warning in the log will be enough?

Reply:
@tmleman I didnt get what you mean by returning 0? in which case would be return 0? when 2 modules belong to different pipelines?

Reply:
return 0 instead of the error code as it has been done so far. Change is for case when modules belong to the same pipeline.